src/middlewares/validation/validateRating.js

```diff
@@ -0,0 +1,23 @@
+import status from '../../config/status';
+
+/**
+ * Validate rating
+ */
+class rating {
+  /**
+   * @param {object} req Request sent to the route
+   * @param {object} res Response from server
+   * @param {object} next If no error continue
+   * @returns {object} Object representing the response returned
+   */
+  static create(req, res, next) {
+    const errors = {};
+    if (req.body && typeof req.body.rating !== 'number') errors.rating = 'rating must be a number';
+    if (req.body.rating < 1) errors.rating = 'rating must be larger than or equal to 1';
```

Comment:
line 16 and 17 can be combined

Reply:
this depends on Gilles's #165412918 Give a rating to an article

Reply:
resolved